src/content/developers/docs/consensus-mechanisms/pos/keys/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Keys in proof-of-stake Ethereum
```

Comment:
Same comment as below. Any reason this can't just be Keys on Ethereum?

Reply:
I think including proof-of-stake makes sense since this page focuses on validator keys. With "Keys on Ethereum" I would assume this to focus more on regular wallet keys. There is a ton of overlap in the topics though, which is why I suggested we potentially expand on this in the future with more content about keys, but this PR is under `consensus-mechanisms/pos/` and I think this makes sense as is.

Reply:
Would it be correct just to call them validator keys?

Reply:
eh - could do, but withdrawal keys are also included on the page.

Reply:
Yeah this is a little tricky.
I tend to think of the content here slightly differently, with "validator keys" consisting of both "signing keys" and "withdrawal keys". The signing keys are "hot" and stay within or available to your validator client.

Only nuance there, is _all_ keys can "sign", so it kinda adds a different naming conflict when trying to explain this stuff.

@jmcook1186 Am I incorrect in this line of thought? Would it be improper to change the "The validator key" section below to "The signing key", and use that terminology instead?

We can then refer to the whole set as "validator keys".

---

Suggestion:
> `title: Keys in proof-of-stake Ethereum`

`title: "Validator keys"`

> `also required a **validator key** and a **withdrawal key**.`

`also required a validator **signing key** and a **withdrawal key**.`

> `### The validator key `

`### The signing key`

And also updating other instances of "validator" key to "signing" where appropriate, or optionally "validator signing keys" occasionally.

Reply:
idk i don't love this bc the withdrawal key is not functionally different from the validator key. "Validator" and "withdrawal" keys separate them by their intended uses - i.e. validator keys are used for validator duties, withdrawal kays are used to withdraw stake - "signing  keys" suggests the alternative is "non-signing keys" which is not true.

Reply:
it's also consistent with previous docs, so I say keep validator/withdrawal for now and update later if necessary?
@minimalsm